### PR TITLE
Import entries in two phases instead of one

### PR DESCRIPTION
**Pull request by neonichu**
This PR is supposed to make it unnecessary to structure the templates so that linked entries are ordered. In order to do that, we first create all entries, then fill them with content in a second phase.

Requires contentful/contentful-management.rb#72

### CODE REVIEW

**Comment by neonichu** [comment]
Unfortunately, this actually doesn't solve the problem because linked entries need to be published before the linking entry can be published. I think the pragmatic solution would be to not make a failure to publish a hard error, so that at least all content can be imported and the correctly ordered publishing can be done manually afterwards if the order is incorrect.

**Comment by neonichu** [comment]
Rebased and made updated recordings for the new specs from master. The blog template didn't specify IDs for some entries, so I updated it on the assumptions that we always want entry IDs to be specified.

File: lib/contentful/bootstrap/templates/base.rb [hunk 1]
```diff
@@ -147,17 +145,37 @@ def create_entries
...
 
-          content_types.each do |content_type|
-            content_type.entries.all.map(&:publish)
+            entry = space.entries.find(e[:id])
+            e.delete(:id)
+            entry.update(e)
+            entry.save
+
+            10.times do |attempt|
+              break if space.entries.find(entry.id).sys[:version] >= 4
```
Comment (dlitvakb) [inline lib/contentful/bootstrap/templates/base.rb]: What does this mean?

Comment (neonichu) [inline lib/contentful/bootstrap/templates/base.rb]: This is there to ensure that we don't run into a race condition

Comment (dlitvakb) [inline lib/contentful/bootstrap/templates/base.rb]: Oh, I understand the reason, but why the version thing? shouldnt be >= 2?

Comment (neonichu) [inline lib/contentful/bootstrap/templates/base.rb]: I haven't investigated in detail, but it ends up being 4 when doing those steps with the CMA gem

**Pull request merged**